try {
    // ...
    // const x;
    throw new Error('hours is empty');
} catch (error) {
    // console.log(error);
} 

// finally{
//     console.log('this code runs no matter what');
// }